docs/core/proto-docs.md

```diff
@@ -90,7 +90,6 @@
     - [Output](#cosmos.bank.v1beta1.Output)
     - [Params](#cosmos.bank.v1beta1.Params)
     - [SendEnabled](#cosmos.bank.v1beta1.SendEnabled)
-    - [Supply](#cosmos.bank.v1beta1.Supply)
```

Comment:
Hmmm why is this diff here?

Reply:
Good question... going to merge master again and will take care of this.